m2m-transfers/V3/REPWRITERSPECS/BANNO.M2MTRANSFERS.V3.POW

```diff
@@ -1998,6 +1998,8 @@ PROCEDURE BUILDSHARELOANLIST
  BNOSLCODE(BNOSHARECODECLUB)=TRUE
```

Comment:
The code looks good. Found some clean up items from before.
On Line 386 in the Set up division please correct the following spelling:
"[*the followowing freq refreences are'

Reply:
@jordan-keenan Corrected.

Reply:
@tkainz Hi Tom, For M2M thanks for improving it so much, and for fixing the typo from a long time ago.
I noticed that sentence that had the typo corrected still has a typo spelling "following" as "followowing" Can you fix that, then I'll approve the review?
"the followowing freq references"
Thanks,
Jordan

Reply:
Thank you @tkainz ! I see that you've changed that, and I've approved the code.